stable/elasticsearch/values.yaml

```diff
@@ -214,6 +220,13 @@ data:
   hooks:  # post-start and pre-stop hooks
     drain:  # drain the node before stopping it and re-integrate it into the cluster after start
       enabled: true
+    # If you need modify the standart drain hooks, set hooks.drain.enabled to false
```

Comment:
That behavior is not intuitive and might look confusing for users. 
"Why should I disable the drain hook if I want the custom one to be executed?"

I suggest to add `data.hooks.drain.pre-stop` and `data.hooks.drain.pre-start` variables with an empty default value. And then render the default body of these scripts in `_helpers.yaml`.  That will allow to keep the backward compatibility with a minimal footprint on `values.yaml`